.gitignore

```diff
@@ -3,3 +3,5 @@
 astrolabe.egg-info/
 results.json
 xunit-output/
+*~
```

Comment:
Drive by fix in separate commit since I almost committed files with these extensions by accident. .iml is a Jet Brains thing FYI.

Reply:
Ack.